lib/ace/ext/modelist.js

```diff
@@ -99,6 +99,7 @@ var supportedModes = {
     JSX:         ["jsx"],
     Julia:       ["jl"],
     LaTeX:       ["tex|latex|ltx|bib"],
+    Lean:        ["lean|hlean"],
```

Comment:
Could you also add a small demo lean file to https://github.com/ajaxorg/ace/tree/master/demo/kitchen-sink/docs, just  a few lines would be fine